data/changelog/ppxlib/2023-06-20-ppxlib-0.30.0.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Ppxlib 0.30.0
+date: "2023-06-20"
+tags: [ppxlib, release]
+changelog: |
+  * Adopt the OCaml Code of Conduct on the repo ([ocaml-ppx/ppxlib#426](https://github.com/ocaml-ppx/ppxlib/pull/426), @pitag-ha)
+  * Clean up misleading attribute hints when declared for proper context. ([ocaml-ppx/ppxlib#425](https://github.com/ocaml-ppx/ppxlib/pull/425), @ceastlund)
+  * Ast_pattern now has ebool, pbool helper, and a new map.([ocaml-ppx/ppxlib#402](https://github.com/ocaml-ppx/ppxlib/pull/402), @Burnleydev1)
+  * multiple errors are now reported in `metaquot`. ([ocaml-ppx/ppxlib#397](https://github.com/ocaml-ppx/ppxlib/pull/397), @Burnleydev1)
+  * Add `Attribute.declare_with_attr_loc` ([ocaml-ppx/ppxlib#396](https://github.com/ocaml-ppx/ppxlib/pull/396), @dvulakh)
+  * Add "ns" and "res" as reserved namespaces([ocaml-ppx/ppxlib#388](https://github.com/ocaml-ppx/ppxlib/pull/388), @davesnx)
+  * Make quoter `let` binding non-recursive ([ocaml-ppx/ppxlib#401](https://github.com/ocaml-ppx/ppxlib/pull/401), @sim642)
+  * Fix failure of 'lift_map_with_context' in traverse by compile-time
+    evaluation of 'fst' and 'snd' ([ocaml-ppx/ppxlib#390](https://github.com/ocaml-ppx/ppxlib/pull/390), @smuenzel)
+  * Driver: Bias the mapping from magic to version towards the current version,
```

Comment:
```suggestion
  * Driver: Bias the mapping from Magic to version towards the current version,
```
Capital M if it's [the company](https://www.magicsoftware.com/) or a proper noun of a tool or something else. 

Also, is "version" a specific tool or company? If so, it needs to have a capital V. 
If not, the sentence doesn't make sense to me. "from magic to version"

If "magic" and "version" are specific things your readers will know but not proper nouns, consider wrapping them in `monospace` or italicized to make them visually stand out as that specific thing rather than generic magic and version. 

It's okay if I don't understand as long as your readers will know what it means.

Reply:
```suggestion
  * Driver: Bias the mapping from `Parsetree` magic numbers to `Parsetree` versions towards the current version,
```

Is that better, @christinerose? None of magic or version is a company. Magic is referring to an identifier number; version is literally a version: the version of the Parrsetree.